docs/pages/linking/into-your-app.mdx

```diff
@@ -0,0 +1,119 @@
+---
+title: Linking into your app
+sidebar_title: Into your app
+description: Learn how to open and handle a URL from other apps based on the scheme for your by creating a deep link.
```

Comment:
```suggestion
description: Learn how to handle an incoming URL in your app
```
Realizing maybe there was some search optimization happening here, so feel free to add back words